DEVELOPMENT.md

```diff
@@ -84,25 +84,54 @@ docker login $REGISTRY
 
 ## Testing
 
-### Using Minikube
+If you want to play with eunomia deployed to Minikube or Minishift, here are some preliminary instructions. They still need a lot of TLC so feel free to send in PRs.
```

Comment:
What "TLC" means?

Reply:
Tender Loving Care, I think. Some further improvements for it to always work in this case. Just left the acronym because I think there's always room for improvement.

Reply:
Ohh, ktx. Never knew. TIL, and all those TLAs tend to often be a bit over my head...